packages/ckeditor5-bookmark/src/editbookmarkcommand.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module bookmark/editbookmarkcommand
+ */
+
+import { Command } from 'ckeditor5/src/core.js';
+import type { Selection, DocumentSelection } from 'ckeditor5/src/engine.js';
+
+/**
+ * The edit bookmark command.
+ *
+ * The command is registered by {@link module:bookmark/bookmarkediting~BookmarkEditing} as `'editBookmark'`.
+ *
+ * To update the bookmarkId of current selected bookmark element, execute the command passing the bookmark id as a parameter:
+ *
+ * ```ts
+ * editor.execute( 'editBookmark', 'newId' );
+ * ```
+ */
+export default class EditBookmarkCommand extends Command {
+	/**
+	 * The value of the `'bookmarkId'` attribute.
+	 *
+	 * @observable
+	 * @readonly
+	 */
+	declare public value: string | undefined;
+
+	/**
+	 * @inheritDoc
+	 */
+	public override refresh(): void {
+		const model = this.editor.model;
+		const selection = model.document.selection;
+
+		this.isEnabled = !!getSelectedBookmark( selection );
+		this.value = this.isEnabled ? getSelectedBookmark( selection )!.getAttribute( 'bookmarkId' ) as string : undefined;
```

Comment:
```suggestion
		const selectedBookmark = getSelectedBookmark( selection );

		this.isEnabled = !!selectedBookmark;
		this.value = selectedBookmark ? selectedBookmark.getAttribute( 'bookmarkId' ) as string : undefined;
```

I would recommend creating a variable to store the value, so there’s no need to call the function twice. This would also allow us to get rid of the `!` as well.